app/models/claims/claim.rb

```diff
@@ -38,14 +41,16 @@ class Claims::Claim < ApplicationRecord
 
   belongs_to :school
   belongs_to :provider
+  belongs_to :claim_window, optional: true
```

Comment:
Can you add a TODO comment, to remove this `optional: true`

Reply:
This follow-up PR removes it on this line: https://github.com/DFE-Digital/itt-mentor-services/pull/868/files#diff-af5fcd79400b248a8923c036a23d85f33328cb0021add465590354c9ffc58902R44

Is that sufficient enough?